log4j-layout-template-json-test/pom.xml

```diff
@@ -189,6 +196,26 @@
       <build>
         <plugins>
 
+          <plugin>
```

Comment:
You don't need this plugin. You just need to add a [jdkToolchain] entry to the `maven-failsafe-plugin` configuration.